src/Tools/DynamoFeatureFlags/FeatureFlagsClient.cs

```diff
@@ -107,7 +107,7 @@ internal FeatureFlagsClient(string userkey, string mobileKey = null, bool testMo
         internal void Init(string mobileKey)
         {
             //start up client.
-            ldClient =  LaunchDarkly.Sdk.Client.LdClient.Init(mobileKey, user,TimeSpan.FromSeconds(5));
+            ldClient =  LaunchDarkly.Sdk.Client.LdClient.Init(mobileKey, LaunchDarkly.Sdk.Client.ConfigurationBuilder.AutoEnvAttributes.Disabled, user, TimeSpan.FromSeconds(5));
```

Comment:
link on ff API changes  https://docs.launchdarkly.com/sdk/client-side/dotnet/migration-3-to-4